jwst/outlier_detection/blot_median.py

```diff
@@ -8,8 +8,8 @@
 """
 import numpy as np
 
-from jwst.resample import gwcs_blot
-from jwst import datamodels
+from ..resample import gwcs_blot
+from .. import datamodels
```

Comment:
I think the form

`from .. import datamodels`

is required for the way we are packaging.